docs/docs/getting-started/concepts.md

```diff
@@ -0,0 +1,641 @@
+      'primaryTextColor': '#fff',
+      'primaryBorderColor': '#231F1B',
+      'lineColor': '#DEDDFF',
+      'secondaryColor': '#BDBAB7',
+      'tertiaryColor': '#BDBAB7'
+    }
+  }
+}%%
+  graph LR
+    style Config fill:#BDBAB7,stroke:#BDBAB7,stroke-width:2px
+    style Partition fill:#BDBAB7,stroke:#BDBAB7,stroke-width:2px
+
+    style Job fill:#BDBAB7,stroke:#BDBAB7,stroke-width:2px
+    style Schedule fill:#BDBAB7,stroke:#BDBAB7,stroke-width:2px
+    style Sensor fill:#BDBAB7,stroke:#BDBAB7,stroke-width:2px
+    style IOManager fill:#BDBAB7,stroke:#BDBAB7,stroke-width:2px
+    style Resource fill:#BDBAB7,stroke:#BDBAB7,stroke-width:2px
+    style AssetCheck fill:#BDBAB7,stroke:#BDBAB7,stroke-width:2px
+    style Definitions fill:#BDBAB7,stroke:#BDBAB7,stroke-width:2px
+
+    AssetCheck -.-> Asset
+    Config -.-> Asset
+    Partition -.-> Asset
+    Resource -.-> Asset
+    IOManager -.-> Asset
+
+    Asset(Asset)
+
+    Asset -.-> Job
+    Asset -.-> Schedule
+    Asset -.-> Sensor
+    Asset ==> Definitions
+```
+
+An `asset` represents a logical unit of data such as a table, dataset, or machine learning model. Assets can have dependencies on other assets, forming the data lineage for your pipelines. As the core abstraction in Dagster, assets can interact with many other Dagster concepts to facilitate certain tasks.
+
+| Concept | Relationship | Required |
+| --- | --- | --- |
+| [asset check](concepts#asset-check) | `asset` may use an `asset check` | No |
+| [config](concepts#config) | `asset` may use a `config` | No |
+| [io manager](concepts#io-manager) | `asset` may use a `io manager` | No |
+| [partition](concepts#partition) | `asset` may use a `partition` | No |
+| [resource](concepts#resource) | `asset` may use a `resource` | No |
+| [job](concepts#job) | `asset` may be used in a `job` | No |
+| [schedule](concepts#schedule) | `asset` may be used in a `schedule` | No |
+| [sensor](concepts#sensor) | `asset` may be used in a `sensor` | No |
+| [definitions](concepts#definitions) | `asset` must be set in a `definitions` to be deployed | Yes |
+
+### Asset Check
+
+```mermaid
+%%{
+  init: {
+    'theme': 'base',
+    'themeVariables': {
+      'primaryColor': '#4F43DD',
+      'primaryTextColor': '#fff',
+      'primaryBorderColor': '#231F1B',
+      'lineColor': '#DEDDFF',
+      'secondaryColor': '#BDBAB7',
+      'tertiaryColor': '#BDBAB7'
+    }
+  }
+}%%
+  graph LR
+    style Asset fill:#BDBAB7,stroke:#BDBAB7,stroke-width:2px
+
+    AssetCheck(Asset Check)
+
+    style Definitions fill:#BDBAB7,stroke:#BDBAB7,stroke-width:2px
+
+    AssetCheck -.-> Asset
+    AssetCheck ==> Definitions
+```
+
+An `asset check` is associated with an `asset` to ensure it meets certain expectations around data quality, freshness or completeness. Asset checks run when the asset is executed and store metadata about the related run and if all the conditions of the check were met.
+
+| Concept | Relationship | Required |
```

Comment:
I would change the first column to "Related concept", and I think you can remove the last column, since that information is already contained in the relationship description (may vs must).